Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest-sorting/ContestSorting.tsx

```diff
@@ -58,21 +59,25 @@ const ContestSorting = ({ onSortClick }: IContestSortingProps) => {
     );
 
     return (
-        <div className={styles.sortingTypeContainer}>
-            <Heading
-              type={HeadingType.small}
-              className={styles.heading}
-            >
-                Sorting
-            </Heading>
-            <List
-              values={possibleSortingTypes}
-              itemFunc={renderSortingItemFunc}
-              orientation={Orientation.horizontal}
-              className={styles.sortTypesList}
-              fullWidth
-            />
-        </div>
+        isEmpty(possibleSortingTypes)
```

Comment:
This seems odd. Shouldnt you render null if something isEmpty and some component when there are values? as in here https://github.com/SoftUni-Internal/OpenJudgeSystem/pull/624/files#diff-24ad9cb85c35d6c6dca2cd0701af2ded2413ef5169d6625d595c3010eeb3a49cR38